interfaces/cython/cantera/ck2cti.py

```diff
@@ -847,6 +847,11 @@ class TransportData(object):
     def __init__(self, label, geometry, wellDepth, collisionDiameter,
                  dipoleMoment, polarizability, zRot, comment=None):
 
+        try:
+            geometry = int(geometry)
+        except ValueError:
+            raise InputParseError("Bad geometry flag '{0}' for species '{1}', must be integer (0, "
+                                  " 1, or 2), not float (0.0, 1.0, 2.0)".format(geometry, label))
```

Comment:
Other errors (e.g. non-numeric strings) will also raise `ValueError`, so the error message shouldn't presume that a float was actually provided.